src/App.tsx

```diff
@@ -1,10 +1,23 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from "react";
+import "./App.css";
+import Header from "./components/Header";
+import { Review } from "./interfaces/review.interface";
+import ListOfReviews from "./components/ListReviews";
+import Background from "./components/Background";
+
+function App(): JSX.Element {
+  const [reviewList, setReviewList] = useState<Review[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
State wise -this could live lower, im not sure what else would use the loading state of the ListOfReviews component. I dont think anything else will use the reviewList either. 

ideally state needs to be as low as possible.